From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Grigoris Pavlakis <grigpavl@ece.auth.gr>
Date: Thu, 9 Jun 2022 00:19:08 +0300
Subject: [PATCH] Change C standard to C99

The originally-set C90 doesn't support the `inline` keyword, causing
weird compile errors because isalnum() and other functions under
ctype.h are actually #define'd as inline.
---
 desktop_version/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/desktop_version/CMakeLists.txt b/desktop_version/CMakeLists.txt
index 7405c12..49d44bb 100644
--- a/desktop_version/CMakeLists.txt
+++ b/desktop_version/CMakeLists.txt
@@ -27,7 +27,7 @@ endif()
 if(${CMAKE_VERSION} VERSION_LESS "3.1.3")
 	message(WARNING "Your CMake version is too old; set -std=c90 -std=c++98 yourself!")
 else()
-	set(CMAKE_C_STANDARD 90)
+	set(CMAKE_C_STANDARD 99)
 	set(CMAKE_C_EXTENSIONS OFF)
 
 	set(CMAKE_CXX_STANDARD 98)
